packages/core/src/types.ts

```diff
@@ -1854,23 +1863,29 @@ export type __ResolvedTypesMetaFrom<T> = T extends StateMachine<
 export type EventOfMachine<TMachine extends AnyStateMachine> =
   TMachine extends StateMachine<any, infer E, any, any, any> ? E : never;
 
-export interface ActorContext<TEvent extends EventObject, TSnapshot> {
+export interface ActorContext<
+  TEvent extends EventObject,
+  TSnapshot,
+  TSystem extends ActorSystem<any> = any
```

Comment:
That's a bad habit 😉  we never should provide `any` as a default type param
```suggestion
  TSystem extends ActorSystem<any>
```